lib/MultiMerchantHeaderSelector.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * ApiException
+ * PHP version 7.4
+ *
+ * @category Class
+ * @package  Gr4vy
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ */
+
+/**
+ * Gr4vy API
+ *
+ * Welcome to the Gr4vy API reference documentation. Our API is still very much a work in product and subject to change.
+ *
+ * The version of the OpenAPI document: 1.1.0-beta
+ * Contact: code@gr4vy.com
+ * Generated by: https://openapi-generator.tech
+ * OpenAPI Generator version: 6.0.0
+ */
+
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+namespace Gr4vy;
+
+use Gr4vy\HeaderSelector as HeaderSelector;
+
+class MultiMerchantHeaderSelector extends HeaderSelector
+{
+    /**
+     * Debug file location (log to STDOUT by default)
+     *
+     * @var string
+     */
+    protected $merchantId;
+
+    /**
+     * Constructor
+     */
```

Comment:
```suggestion
class MerchantAccountHeaderSelector extends HeaderSelector
{
    /**
     * Debug file location (log to STDOUT by default)
     *
     * @var string
     */
    protected $merchantAccountID;

    /**
     * Constructor
     */
```

Can we keep this consistent and use merchant account ID?